app/src/main/java/com/github/se/travelpouch/model/TravelContainer.kt

```diff
@@ -124,3 +126,109 @@ data class Location(
     require(name.isNotBlank()) { "Location name cannot be blank" }
   }
 }
+
+/*
+ * Mock data generator for TravelContainer.
+ */
+object TravelContainerMock {
```

Comment:
I think this object is used to avoid repeting the same steps, in the tests, to create a new travel container. For me it is a very good idea.